index.html

```diff
@@ -133,7 +133,7 @@
             // router: { key: { document: 'uid', }, }, // uncomment to enable navigation by UID (see WEBUI-14)
             /* selection: { // uncomment to enable select all items in listings
               selectAllEnabled: true
-            }, */
+            },*/
```

Comment:
please, add the missing space
```suggestion
            }, */
```